Tax 1.2% in all transactions

```diff
@@ -0,0 +1,12 @@
+
+[{
+“subspace”: “treasury”,
```

Comment:
These `“` should be replaced with `"`.

Reply:
I think its the way you see on git 
Its ok